test/Fail/DuplicateExecutable.err

```diff
@@ -1,3 +1,4 @@
+: error: [LibraryError]
```

Comment:
This looks weird, why is there no file name or range?

Reply:
Done.